src/app/shared/loader/cow-loader.component.ts

```diff
@@ -0,0 +1,17 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'app-cow-loader',
+  standalone: true,
+  imports: [CommonModule],
+  template: `
+    <div class="loader">
+      <img src="/assets/logo.png" alt="ładowanie, proszę czekać" />
+      <p>Ładowanie...</p>
+    </div>
+  `,
+  styleUrls: ['./style.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CowLoaderComponent {}
```

Comment:
To chyba nie jest komponent z tego taska, nie powinno się tu znaleźć

Reply:
Masz rację, przez przypadek wpadło z poprzedniego taska...